main/src/vtr_gui/vtr_gui/vtr-gui/src/components/goal/GoalManager.js

```diff
@@ -54,6 +56,8 @@ class GoalManager extends React.Component {
     } = this.props;
     const { goal_panel_open } = this.state;
 
+<<<<<<< HEAD
```

Comment:
This was missed in the merge and should be deleted.

Reply:
Fixed